assets/blocks/progress-bar/style.scss

```diff
@@ -0,0 +1,37 @@
+.wp-block-sensei-lms-progress-heading {
+	display: flex;
+
+	&__lessons {
+		font-weight: bold;
+		flex-basis: 10%;
```

Comment:
Not sure if any layout is needed here, this can be just inline text. Even if it's not, these sizes don't really do it any justice:
<img width="617" alt="image" src="https://user-images.githubusercontent.com/176949/95483180-35a65a80-098f-11eb-8511-2feed4a9761a.png">

Reply:
I was thinking the Continue button when made this a flexbox but we can leave it out for now: 408561569ece8695446c7c83c9389499110bdd1b

Reply:
We are probably skipping that for now, and the ideal goal there would be to use the Columns block with these in it as a template.